lib/spack/spack/test/database.py

```diff
@@ -353,3 +359,19 @@ def fail_while_writing():
     # reload DB and make sure cmake was not written.
     with install_db.read_transaction():
         assert install_db.query('cmake', installed=any) == []
+
+
+def test_external_entries_in_db(database):
+    install_db = database.mock.db
+    rec = install_db.get_record('mpileaks ^zmpi')
```

Comment:
can you put empty lines before these.. just to make it more visually obvious how many cases there are in this test? 😄